portfolio/src/main/java/com/google/sps/controller/SimpleHtmlServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.controller;
+
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+
+@WebServlet(name = "simple-html",
+            urlPatterns = {"/simple-html"})
+public class SimpleHtmlServlet extends HttpServlet {
+
+    public void doGet(HttpServletRequest request,
+                      HttpServletResponse response) throws
+                                                                                ServletException,
```

Comment:
Is this indentation/spacing intentional?

Reply:
I think this might be a bad habit, and I will change this